ghcup-prereleases-0.0.7.yaml

```diff
@@ -952,6 +952,121 @@ ghcupDownloads:
+              dlOutput: ghc-9.8.0.20230809-x86_64-linux-ubuntu18_04.tar.xz
+              dlSubdir: ghc-9.8.0.20230809-x86_64-unknown-linux
+              dlUri: https://downloads.haskell.org/~ghc/9.8.0.20230809/ghc-9.8.0.20230809-x86_64-ubuntu18_04-linux.tar.xz
+            '>= 20': &ghc-981alpha2-ubuntu20_04
+              dlHash: dad1796ddcec29b3bb090d8dd97fb4e95d29d4d1be0adbda1e1e55829eddb074
+              dlOutput: ghc-9.8.0.20230809-x86_64-linux-ubuntu20_04.tar.xz
+              dlSubdir: ghc-9.8.0.20230809-x86_64-unknown-linux
+              dlUri: https://downloads.haskell.org/~ghc/9.8.0.20230809/ghc-9.8.0.20230809-x86_64-ubuntu20_04-linux.tar.xz
+          Linux_RedHat:
+            unknown_versioning: *ghc-981alpha2-centos7
+          Linux_Ubuntu:
+            ( >= 16 && < 19 ): *ghc-981alpha2-ubuntu18_04
+            unknown_versioning: *ghc-981alpha2-ubuntu20_04
+          Linux_UnknownLinux:
+            unknown_versioning:
+              dlHash: f49639a2a36e773e2293954b267f9e6d8cce0d8d800cd36863b0ad10ec6b9304
+              dlOutput: ghc-9.8.0.20230809-x86_64-linux-rocky8.tar.xz
+              dlSubdir: ghc-9.8.0.20230809-x86_64-unknown-linux
+              dlUri: https://downloads.haskell.org/~ghc/9.8.0.20230809/ghc-9.8.0.20230809-x86_64-rocky8-linux.tar.xz
+          Windows:
+            unknown_versioning:
+              dlHash: eadf5f37adf2ed127c896035908a1b25c0a9cd8d1b2706feb9cde44d453fe911
+              dlOutput: ghc-9.8.0.20230809-x86_64-windows.tar.xz
+              dlSubdir: ghc-9.8.0.20230809-x86_64-unknown-mingw32
+              dlUri: https://downloads.haskell.org/~ghc/9.8.0.20230809/ghc-9.8.0.20230809-x86_64-unknown-mingw32.tar.xz
+        A_ARM64:
+          Darwin:
+            unknown_versioning:
+              dlHash: dcd2c76c790e0933c695d19df262eed179719eb9369e40f704c17d83ad468bb0
+              dlOutput: ghc-9.8.0.20230809-aarch64-darwin.tar.xz
+              dlSubdir: ghc-9.8.0.20230809-aarch64-apple-darwin
+              dlUri: https://downloads.haskell.org/~ghc/9.8.0.20230809/ghc-9.8.0.20230809-aarch64-apple-darwin.tar.xz
+          Linux_UnknownLinux:
+            unknown_versioning:
+              dlHash: fed0c4372f0288d62e41b4af379a8c4d18ad2f9792973246f3650395911361e9
+              dlOutput: ghc-9.8.0.20230809-aarch64-linux-deb10.tar.xz
+              dlSubdir: ghc-9.8.0.20230809-aarch64-unknown-linux
+              dlUri: https://downloads.haskell.org/~ghc/9.8.0.20230809/ghc-9.8.0.20230809-aarch64-deb10-linux.tar.xz
+      viChangeLog: https://downloads.haskell.org/~ghc/9.8.0.20230809/docs/users_guide/9.8.1-notes.html
+      viReleaseDay: '2023-03-09'
+      viSourceDL:
+        dlHash: ecef10e9796082eaf3f18ece67fe5fcfedfd0baa77d34c1f772e37aabda3fa1c
+        dlOutput: ghc-9.8.0.20230809-src.tar.xz
+        dlSubdir: ghc-9.8.0.20230809
+        dlUri: https://downloads.haskell.org/~ghc/9.8.0.20230809/ghc-9.8.0.20230809-src.tar.xz
+      viTags:
+      - Prerelease
```

Comment:
@bgamari this is wrong too. It needs to be `LatestPrerelease` and the old one changed to `Prerelease`.